resources/public/contracts/src/strings.sol

```diff
@@ -0,0 +1,716 @@
+/*
```

Comment:
Note that I eventually made only one small change against the original repository (the copypasted version in this repo was older than the newest one in the original repo). The original repo has lot of tests and presumably the owner ran them before merging.

It would be nice if I ran the tests myself on my small update of the repo, but they use some dapp crap which in turn tries to force nix on me or something, so I left it for now.

Reply:
I created an issue to not forget this: https://github.com/vacuumlabs/name-bazaar/issues/10